routes/mapper.py

```diff
@@ -164,21 +164,21 @@ def __init__(self, obj, resource_name=None, collection_name=None,
 
     def connect(self, *args, **kwargs):
         newkargs = {}
-        # newargs = args
-        routename, path = args
+        routename = args[0]
+        path = args[1] if len(args) > 1 else None
```

Comment:
Using `*args` is overengineering. It's simpler to drop them. This will make code more straightforward.